packages/compiler/core/checker.ts

```diff
@@ -732,9 +732,12 @@ export function createChecker(program: Program): Checker {
 
   function resolveIdentifierInTable(
     node: IdentifierNode,
-    table: SymbolTable,
+    table: SymbolTable | undefined,
     resolveDecorator = false
   ) {
+    if (!table) {
+      return undefined;
+    }
```

Comment:
I can't find the exact repro for when this is undefined, but this happened twice to me while ad-hoc testing colorization with particularly malformed source.